GrocifyApp.UI/src/index.css

```diff
@@ -22,14 +22,6 @@ a:hover {
   color: #535bf2;
 }
 
-body {
-  margin: 0;
-  display: flex;
-  place-items: center;
-  min-width: 320px;
-  min-height: 100vh;
-}
-
 h1 {
```

Comment:
do we need h1?

Reply:
don't know, this file was created with react app, I've just removed the css that had conflicts with my own css